examples/methods/methods.go

```diff
@@ -6,19 +6,28 @@ import "fmt"
 
 type rect struct {
 	width, height int
+	label         string
 }
 
-// This `area` method has a _receiver type_ of `*rect`.
+// This `area` method has a _pointer receiver type_ `*rect`.
 func (r *rect) area() int {
 	return r.width * r.height
 }
 
 // Methods can be defined for either pointer or value
-// receiver types. Here's an example of a value receiver.
+// receiver types. This method `perim` of _value receiver type_
```

Comment:
How about:

 ... Here's an example of a value receiver, which takes the `rect` struct by value instead of by reference.